ui/css/perf.css

```diff
@@ -449,3 +449,23 @@ td.alert-confidence {
 .alert-strike {
     text-decoration: line-through;
 }
+
+.compare-table-tooltip .tooltip-inner {
+  max-width: 390px;
+}
+
+.tool-tip-table {
+  border: 1px solid white;
+  height: 20px;
+  margin-bottom: 5px;
+}
+.tool-tip-table .value-column {
+  width: 50px;
+  border: 1px solid white;
+  align: center;
+}
+
+.tool-tip-table .distribution-column{
+  width: 280px;
```

Comment:
The tooltips still seem kind of wide to me:

<img width="412" alt="screen shot 2017-06-12 at 10 56 51 am" src="https://user-images.githubusercontent.com/20569/27040072-f01eaf10-4f5d-11e7-9ed2-a9be6ed92ac3.png">

Maybe you also need to adjust `compare-table-tooltip` above? I would say we would only want this to be 66%-75% (your discretion) as big as it is now